coops-ui/tests/unit/Certify.spec.ts

```diff
@@ -0,0 +1,172 @@
+//
```

Comment:
Is this header required? If so, please create a ticket to update all the other files. Thx!

Reply:
Required? No idea. Probably a decision for the team to make.

Reply:
It is required. Opened #1291 as tech debt.